frontend/packages/kitconcept-core/package.json

```diff
@@ -38,25 +38,24 @@
     "@mbarde/volto-image-crop-widget",
     "@plonegovbr/volto-social-media",
     "@plone-collective/volto-authomatic",
-    "volto-form-block",
```

Comment:
Let's remove it until we don't have Rob's final version. I guess it will happen in the sprint where the RT guys will be there too. 

@robgietema btw, anything that is holding up to release the final version of the package?